.github/dependabot.yml

```diff
@@ -5,6 +5,19 @@ updates:
     schedule:
       interval: "weekly"
     open-pull-requests-limit: 10
+    groups:
+      golang-x:
+        patterns:
+          - "golang.org/x/*"
+      k8s:
+        patterns:
+          - "k8s.io/*"
+      moby-sys:
+        patterns:
+          - "github.com/moby/sys/*"
```

Comment:
These don't necessarily have to be updated in tandem, as they're independent modules; does this also mean they're always updated in a single commit?

Reply:
Let's see how it works, and maybe we can remove this line later